DataConnectors/CEF/cef_troubleshoot.py

```diff
@@ -383,6 +384,23 @@ def check_oms_agent_status():
     else:
         return True
 
+def check_omsagent_new_version(omsagent_plugin_securiy_config):
+        '''
+        Checking if the OMS agent has the updated configuration:
+        :param omsagent_plugin_security_config: path to the file containing the updated configuration
+        :return: True if the configuration is updated, false otherwise
+        '''
+        grep = subprocess.Popen(["grep", "-i", "return ident", oms_agent_plugin_securiy_config], stdout=subprocess.PIPE)

```

Comment:
Maybe use the full line? "return indent" is something that could be used in the future for other cases.
Also add to the consts